pyjelly/integrations/rdflib/parse.py

```diff
@@ -366,97 +312,80 @@ def graphs_from_jelly(
     options, frames = get_options_and_frames(inp)
 
     if options.stream_types.logical_type == jelly.LOGICAL_STREAM_TYPE_FLAT_TRIPLES:
-        yield parse_flat_triples_stream(frames=frames, options=options, store=store)
+        yield parse_flat_triples_stream(frames=frames, options=options, store=store)  # type: ignore[call-arg]
         return
 
     if options.stream_types.logical_type == jelly.LOGICAL_STREAM_TYPE_FLAT_QUADS:
-        yield parse_flat_quads_stream(frames=frames, options=options, store=store)
+        yield parse_flat_quads_stream(frames=frames, options=options, store=store)  # type: ignore[call-arg]
         return
 
     if options.stream_types.logical_type == jelly.LOGICAL_STREAM_TYPE_GRAPHS:
-        yield from parse_graph_stream(frames=frames, options=options, store=store)
+        yield from parse_graph_stream(frames=frames, options=options, store=store)  # type: ignore[call-arg]
         return
 
     logical_type_name = jelly.LogicalStreamType.Name(options.stream_types.logical_type)
     msg = f"the stream type {logical_type_name} is not supported "
     raise NotImplementedError(msg)
 
 
-def graph_from_jelly(
+def parse_jelly_flat(
     inp: IO[bytes],
-    store: Store | str = "default",
-    identifier: str | None = None,
+    sink: Graph,
+    lambda_graph: Callable[[Any, Any], Graph],
+    lambda_dataset: Callable[[Any], Dataset],
```

Comment:
Suggest renaming to `graph_factory` or `create_graph` or something like that.

The lambdas should have no arguments....